modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1969,6 +1969,11 @@
     <string name="onboarding_coming_from_another_app">Coming from another app? Import your podcasts and get listening. You can always do this later in settings.</string>
     <string name="onboarding_import_from_castbox">Import from Castbox</string>
     <string name="onboarding_import_from_other_apps">Import from other apps</string>
+    <string name="onboarding_import_from_url">Import from URL</string>
+    <string name="onboarding_import_from_url_description">Import your podcasts from an OPML file using a URL</string>
```

Comment:
```suggestion
    <string name="onboarding_import_from_url_description">Import your podcasts from an OPML file using a URL.</string>
```

iOS doesn't have a full stop either but in other areas of the app we do. This is just a suggestion, feel free to ignore it.